media-plugins/mythplugins/mythplugins-31.0.ebuild

```diff
@@ -0,0 +1,163 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# Change this when bumping to a new major version.
+MAJOR_PV=31
+
+PYTHON_COMPAT=( python2_7 python3_{6,7} )
```

Comment:
Aaaany chance you'd remove python2_7 altogether?

Reply:
Upstream supports python2_7 and one plugin has not been converted yet.